project_parameters.Config.yaml

```diff
@@ -51,6 +51,6 @@ use_condition_split_filter_object: "YES" # set for 04_run_filter_object.Rmd
 print_pdf_filter_object: "NO" # set for 04_run_filter_object.Rmd
 use_scDblFinder_filtering_filter_object: "NO" # set for 04_run_filter_object.Rmd
```

Comment:
The use_scDblFinder_filtering_filter_object param is duplicated in the yaml. One of the lines needs to be removed for lsf script to run.

Reply:
Fixed here 0149196e6430781d9c778bcc9f352b65351ec9ad.